tools/ci/src/commands/subcommands/bench_check_command.rs

```diff
@@ -0,0 +1,17 @@
+use crate::commands::{Flag, Prepare, PreparedCommand};
+use argh::FromArgs;
+use xshell::cmd;
+
+/// Checks that the benches compile.
+#[derive(FromArgs, Default)]
+#[argh(subcommand, name = "bench-check")]
+pub(crate) struct BenchCheckCommand {}
```

Comment:
```suggestion
pub(crate) struct BenchCheckCommand;
```

If the subcommands don't contain any fields, they can be converted to unit structs.

Beyond that, this looks good to go! (Don't worry about CI, btw, it's failing because of our dependency on Taffy.)

Reply:
I believe `argh` disallows unit structs. I can double check, though

Reply:
I tested it, your correct! Good to know for the future